### PR TITLE
Changed offset for some locations

### PR DESCRIPTION
**Pull request by DenisLapi**

### CODE REVIEW

File: timezones.json [hunk 1]
```diff
@@ -39,7 +39,7 @@
   {
     "value": "Alaskan Standard Time",
     "abbr": "AKDT",
-    "offset": -8,
+    "offset": -9,
```
Comment (dmfilipenko) [inline timezones.json]: https://www.timeanddate.com/time/zones/akdt

File: timezones.json [hunk 2]
```diff
@@ -53,7 +53,7 @@
   {
     "value": "Pacific Standard Time (Mexico)",
     "abbr": "PDT",
-    "offset": -7,
+    "offset": -8,
```
Comment (dmfilipenko) [inline timezones.json]: https://www.timeanddate.com/time/zones/pdt

Comment (dmfilipenko) [reply timezones.json]: And so on. Why are you proposing this changes?

Comment (DenisLapi) [reply timezones.json]: @dmfilipenko `offset` property values doesn't match the number from `text` property. For example:
http://prntscr.com/i792ph

Comment (dmfilipenko) [reply timezones.json]: because `isdst` time

Comment (dmfilipenko) [reply timezones.json]: Closing due unneeded.

**Pull request closed without merging**